### PR TITLE
Infra: wireit integration

### PR DESCRIPTION
**Pull request by ryan-roemer**
Try it out!

```sh
$ pnpm install
$ pnpm build
$ pnpm check
```

or

```sh
$ pnpm check --watch
```

... and just do dev work.

Core work:

- Add wireit integration for all common tasks that can/should be parallelized / part of a dependency graph.
    - Use `nps` to provide support for root tasks.
- Simplify out developer workflow in `CONTRIBUTING.md`
- Move common dependencies (TS, Babel, Webpack) all to root package.json
- Remove prettier rules from eslint (make _only_ prettier fail for style failures)

Other unrelated things:

- Various small fixes to `create-spectacle` from move from `spectacle-cli`
- Fix changesets ci to add `pnpm install --no-frozen-lockfile`
- Fix various file links in docs that have now changed.

Work log:

- [x] `start*` (all examples)
- [x] `types` build + check
- [x] `check`
- [x] Update `CONTRIBUTING.md` with optimized suggested developer workflow
- [x] `test`
- [x] `lint`
- [x] `prettier`
- [x] `build` (es, lib, dist)
- [x] `clean*` (see victory)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,25 +5,30 @@
...
 </a>
 </p>
 
-Looking for a quick preview of what you can do with Spectacle? Check out our Live Demo deck
-[here](https://raw.githack.com/FormidableLabs/spectacle/main/examples/one-page.html).
+## Getting Started
+
+Welcome to our monorepo project, housing the core [`spectacle` package](https://github.com/FormidableLabs/spectacle/blob/main/packages/spectacle/README.md) and related tools and examples.
+
+Come learn more at https://www.formidable.com/open-source/spectacle!
```
Comment (gksander) [inline README.md:17]: Should we make this an explicit anchor tag with `[...](...)`?

File: package-scripts.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      'cross-env BABEL_ENV=es nps "babel:pkg:base src --out-dir es"',
+    'babel:pkg:lib:cjs':
+      'cross-env BABEL_ENV=commonjs nps "babel:pkg:base src --out-dir lib"',
+
+    // - Webpack
+    webpack: 'webpack',
+
+    // Quality.
+    // - Format
+    // TODO(wireit): IMPLEMENT
```
Comment (gksander) [inline package-scripts.js:35]: Seems like this is TODONE?

**Review by gksander — APPROVED** [review]
Nice, looks good to me! Dev workflow feels really nice!

File: README.md (hunk 1 above)
Comment (ryan-roemer) [reply README.md]: Yep!

File: package-scripts.js (hunk 2 above)
Comment (ryan-roemer) [reply package-scripts.js]: Yep!

**Review by carlos-kelly — APPROVED** [review]

**Pull request merged**